apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2.Tests/BigtableVersionTest.cs

```diff
@@ -67,7 +67,7 @@ public void RoundTripLong(long value)
         }
 
         [Theory]
-        [InlineData("BigtableVersion: 63643449600000000", "2017-10-13Z")]
+        [InlineData("BigtableVersion: 1507852800000000", "2017-10-13Z")]
         public void FormattingDateTime(string expectedText, string dateTimeText)
         {
             var dateTime = DateTime.Parse(dateTimeText).ToUniversalTime();
```

Comment:
Hadn't spotted this before. Better to use `DateTime.ParseExact`, specifying the invariant culture and `DateTimeStyles.AssumeUniversal | DateTimeStyles.AdjustToUniversal`

Reply:
Thanks. I used "u" as the format, but let me know if there's something better to use.

Reply:
I'd just be explicit: "yyyy-MM-dd" (with the invariant culture) and remove "Z" from the input strings. That's probably simpler for most devs to read than remembering what "u" means.

Reply:
`ParseExact` requires a format string though.

Reply:
Oh, I misunderstood. Sorry